test/Microsoft.ML.Tests/Scenarios/SentimentPredictionTests.cs

```diff
@@ -65,16 +65,16 @@ public void TrainAndPredictSentimentModelTest()
             var evaluator = new BinaryClassificationEvaluator();
             BinaryClassificationMetrics metrics = evaluator.Evaluate(model, testData);
 
-            Assert.Equal(.7222, metrics.Accuracy, 4);
-            Assert.Equal(.9643, metrics.Auc, 1);
-            Assert.Equal(.96, metrics.Auprc, 2);
+            Assert.Equal(.5556, metrics.Accuracy, 4);
```

Comment:
@justinormont - the quality of this model went really far down with this change.  Note the confusion matrix below - there was only 1/9 negative sentences predicted correctly.